packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleVideoViewer/SingleVideoViewerContainer.js

```diff
@@ -23,270 +23,182 @@ const DrawingLayer = styled.div`
-                  {/* {title?.id && title?.text && (
-                  <title id={title.id}>{title.text}</title>
-                )} */}
-                  <g ref={this.transformLayer} transform={transform}>
-                    {enableInteractionLayer && (
-                      <InteractionLayer
-                        scale={scale}
-                        played={played}
-                        duration={duration}
-                        width={naturalWidth}
-                        height={naturalHeight}
-                      />
-                    )}
-                  </g>
-                </svg>
-              </SVGContext.Provider>
-            </Box>
-          </DrawingLayer>
-        </ScreenContainer>
-        <VideoController
-          isPlaying={isPlaying}
-          played={played}
-          playbackRate={playbackRate}
-          duration={duration}
-          onPlayPause={this.handlePlayPause}
-          onSpeedChange={this.handleSpeedChange}
-          onSliderMouseUp={this.handleSliderMouseUp}
-          onSliderMouseDown={this.handleSliderMouseDown}
-          onSliderChange={this.handleSliderChange}
-        />
-      </>
-    )
-  }
+    playerRef?.current.seekTo(played)
+  }
+
+  const handlePlayerError = (error) => {
+    onError(error)
+  }
+
+  const canvas = transformLayer?.current
+  const interactionLayerScale = clientWidth / videoWidth
+
+  return (
+    <>
+      {videoSrc
+        ? (
+          <SubjectContainer>
+            <SingleVideoViewer
+              isPlaying={isPlaying}
+              onDuration={handleVideoDuration}
+              onEnded={handleVideoEnded}
+              onError={handlePlayerError}
+              onReactPlayerReady={onReactPlayerReady}
+              onProgress={handleVideoProgress}
+              playbackRate={playbackRate}
+              playerRef={playerRef}
+              url={videoSrc}
+            />
+            {/* Drawing Layer */}
+            <DrawingLayer>
+              <Box overflow='hidden'>
+                <SVGContext.Provider value={{ canvas }}>
+                  <svg
+                    ref={interactionLayerSVG}
+                    focusable
+                    onKeyDown={onKeyDown}
+                    tabIndex={0}
+                    viewBox={`0 0 ${videoWidth} ${videoHeight}`}
+                    xmlns='http://www.w3.org/2000/svg'
+                  >
+                    <g ref={transformLayer} transform=''>
+                      {enableInteractionLayer && (
+                        <InteractionLayer
+                          scale={interactionLayerScale}
+                          duration={duration}
+                          height={videoHeight}
+                          played={timeStamp}
+                          width={videoWidth}
+                        />
+                      )}
+                    </g>
+                  </svg>
+                </SVGContext.Provider>
+              </Box>
+            </DrawingLayer>
```

Comment:
Should the drawing layer be rendered if there isn't a drawing task in the workflow?